examples/tests/whiteboard.py

```diff
@@ -19,7 +18,6 @@ class WhiteBoard(Test):
     """
 
     def test(self):
-        os.chdir(self.datadir)    # Just to stress out Avocado
```

Comment:
There is a typo in commit message (minor thing). Anyway the whole whiteboard test is selftest-oriented so IMO it should be moved to `selftests` and replaced with simpler one written with users in mind, what do you think? (not including this patch would help in remembering of the stress)